runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -1420,6 +1422,13 @@ public CorfuRuntime enableTls(String keyStore, String ksPasswordFile, String tru
         parameters.trustStore = trustStore;
         parameters.tsPasswordFile = tsPasswordFile;
         parameters.tlsEnabled = true;
+
+        // Start file watcher on Ssl certs if it is not initialized
+        if(!sslCertWatcher.isPresent()){
+            log.info("enableTls: Initializing sslCertWatcher.");
+            sslCertWatcher = getSslCertWatcher();
+        }
+
```

Comment:
Spawning a thread from the field setter is a little unexpected and duplicates some of the code that's in one of the constructers. I suggest moving it to `connect` where other resources are created at that time.